### ISSUE TITLE
many communities with a single node

### ISSUE THREAD

**Issue by seinecle** [vtraag/gephi-leiden-plugin#2]
The algo finds many communities which are made of just one node, even when these nodes are not isolated. That is not very useful and actually on large networks, this creates hundreds of communities which make the results impractical to process further.
Suggestion: it could be left to the user not to create communities that have a member count < n?

**Comment by vtraag** [comment]
Yes, that may be one possibility. I am not sure if Gephi offers the possibility of not assigning a node to a cluster, I'll check it out. We are usually also not interested in the small communities, and sometimes actually force these small communities back into larger communities. I'll think about how to proceed.

But regardless of the small communities, Gephi should not become slow as a result of it. Is it possible for you to share your data with me, so that I can replicate the problem? Feel free to mail them to me instead of posting them here.

**Comment by vtraag** [comment]
I tried to test with the power grid example provided by default by Gephi. But also when the Leiden algorithm returns a high number of (small) communities, I don't see any noticeable slow down of Gephi. Could you please describe how I should replicate the problem? Thanks!

**Comment by NETfrix** [comment]
This might happen when the result is "Quality - NaN".
I found that when one of the column headers in the Edges Table is "kind" (it happens when using the Twitter API) - you get "Quality - NaN" and then n=#communities. I solved it by changing the header to "label" (for example).

**Comment by vtraag** [comment]
@NETfrix , this is presumably a separate problem, I'll follow up on it in a separate issue.
